packages/gatsby/cache-dir/gatsby-browser-entry.js

```diff
@@ -1,97 +1,7 @@
-import React from "react"
-import PropTypes from "prop-types"
 import loader from "./loader"
 
 const prefetchPathname = loader.enqueue
 
-const StaticQueryContext = React.createContext({})
```

Comment:
Diff between this file and new file is nearly identical, just copied things to another file: https://www.diffchecker.com/UxZajJ1m